### ISSUE TITLE
Configure Slots function on servers

### ISSUE THREAD

**Issue by VermillionX** [TechReborn/TechReborn#1406]
If a player on a server running the latest version of TechReborn uses the "Configure Slots" function, it disconnects everyone on the server except the person configuring the slots and will continue to do so until they stop configuring slots.
Additionally, there are many machines that can't have their slots configured. They lack the button to do so, and now can only be operated manually (Scrapboxinator, EBF, etc...).

**Comment by knie1988** [comment]
Same here. All machines -_-

Forge Minecraft 1.12.2
TechReborn 1.12.2-2.11.1.420
RebornCore 1.12.2-3.6.1.183
Applied Energistics 2-rv5-stable-4
Sponge (Forge)  1.12.2-2555-7.1.0-BETA-2825

**Comment by modmuss50** [comment]
Is there anything unusual in the client log? Or even possibly on the server log?

**Comment by knie1988** [comment]
The normal Log is not helpfull, all other player disconnects with "lost connection" error:
```
[16:10:50] [Server thread/INFO]: UserXYZ lost connection: Disconnected
[16:10:50] [Server thread/INFO]: UserXYZ left the game
```

I will watch out today the FML Logfiles and will post an snippet.

**Comment by knie1988** [comment]
So, here is my information:
Players who are further away lost connection, are there nearby (up to about 100 blocks) they do not have the problem.
Client-Logfile:
https://pastebin.com/f85K8ZXr

**Comment by modmuss50** [comment]
Ah yeah, thats helpful. Thanks. I wont be able to fix this till the weekend but it should be an easy fix.

**Comment by modmuss50** [comment]
Fixed with: RebornCore-1.12.2-3.6.2.187
